css/custom.css

```diff
@@ -451,6 +443,10 @@ h2.swal2-title {
   background: #47a883;
 }
 
+.stats-btn ul.stats-ul li:hover{
+  background-color: rgba(0,0,0,0.2);
```

Comment:
You add hover properties. Also please see the guidelines or change them: https://github.com/fossasia/labyrinth/blob/master/UI_Identity.md They are useful to give a uniform experience for all game sites. What do you think? Since you are working s much on the UI, you have an overview about these things. What do you think?

Reply:
Yes @niccokunzmann we can go with uniform experience throughout the site.
I thought making a little contrast for the hover property in the main menu and the tabs within the menu.
So shall I change it to the hover colour as per the guidelines or this is good?

Reply:
I will merge it, we can have a look. If you think, it looks better than what the documentation says, please change the documentation.

Reply:
Okay I'll make changes in the documentation for the hover colour to be used for the tabs within the menu.